### DISCUSSION TITLE
Getting the size of a pane (rows x cols) with Wezterm

### DISCUSSION THREAD

**Discussion by j-xella** [wezterm/wezterm#4027] [Q&A] [answered]
Is it possible to get the size of the panes (rows and columns), purely by wezterm means. Possibly some fancy lua call from debug overlay?

**Comment by nekowinston** [comment, marked as answer]
In the debug overlay it'd be `window:active_pane():get_dimensions()`, in the events emitted by the Window object, it's either the `pane` argument or the active `pane` in `panes[]`.
